katsdpcontroller/product_controller.py

```diff
@@ -1172,8 +1171,9 @@ async def _launch_telstate(self) -> katsdptelstate.aio.TelescopeState:
         # connect to telstate store
         self.telstate_endpoint = '{}:{}'.format(self.telstate_node.host,
                                                 self.telstate_node.ports['telstate'])
-        redis_client = await aioredis.create_redis_pool(f'redis://{self.telstate_endpoint}')
-        telstate_backend = katsdptelstate.aio.redis.RedisBackend(redis_client)
+        telstate_backend = await katsdptelstate.aio.redis.RedisBackend.from_url(
```

Comment:
Why construct the backend at all? Why not just pass the URL to `TelescopeState`?

Reply:
The aio version of TelescopeState can only be constructed from a Backend - something I did to avoid the complex overloading in the synchronous version. That also solves the problem that a constructor cannot be asynchronous.

Reply:
Right, thanks!